guides/micronaut-rabbitmq-rpc/bookcatalogue/java/src/main/java/example/micronaut/Book.java

```diff
@@ -1,17 +1,17 @@
 package example.micronaut;
 
+import io.micronaut.core.annotation.Creator;
 import io.micronaut.core.annotation.Introspected;
 
 import java.util.Objects;
 
 @Introspected
 public class Book {
-    private String isbn;
-    private String name;
 
-    public Book() {
-    }
+    private final String isbn;
+    private final String name;
 
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.